### DISCUSSION TITLE
Cannot login on Proton mail

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/Mull-Fenix#179] [Q&A]
As title,I cannot login to protonmail account using mull browser.After filling cred. and hitting login loading icon appear, starts rotating, slugs, disappear and never appear again.
I tried multiple times, waited for upto 10 minutes,Multiple network switches but nothing changed.
That loading icon disappear and do not appear again.
I do not get this issue in bromite,fennec or tor on same device so probably issue with browser.
Probably issue with about:config adjustments so I want to know what might be breaking site and FIX for it.

**Comment by SkewedZeppelin** [comment]
They either use some proof of work captcha or complex crypto that takes forever to complete when the JIT is disabled.
You can see the same issue if you set Tor Browser's security slider to Safer.
You can re-enable the JIT by changing the following preferences in about:config to true
```
javascript.options.ion
javascript.options.baselinejit
```

Also you should not be using Bromite as it hasn't been updated since December of 2022, consider replacing it with Cromite.
